anitya/tests/lib/backends/test_init.py

```diff
@@ -24,19 +24,22 @@
 from __future__ import absolute_import, unicode_literals
 
 import re
-import unittest
+import six.moves.urllib.request as urllib2
 
+import unittest
```

Comment:
Following the PEP 8 rules, unittest should stay up in the top group and six should move to this group

Reply:
Didn't know this